hphp/test/tools/import_zend_test.py

```diff
@@ -1366,6 +1366,8 @@ def parse_headers(zend):
         test = test.replace('EmptyFile.txt', 'md5_EmptyFile.txt')
     if 'ext/standard/tests/strings/lcfirst.php' in full_dest_filename:
         test = test.replace('dummy.txt', 'dummy-lcfirst.txt')
+    if 'ext/standard/tests/strings/strncmp_variation6.php' in full_dest_filename:
+        test = re.sub(r"^var_dump\( strncmp\(\$str1, \$str2, 8\) \)', 'var_dump( strncmp($str1, $str2, 8) > 0)', test)
```

Comment:
This doesn't have matched quotes.  Can you also confirm this functions properly?